src/SessionManager/ProgramSettings.cc

```diff
@@ -54,7 +54,8 @@ ProgramSettings::ProgramSettings(int argc, char** argv) {
         ("frontend_folder", "Set folder to serve frontend files from", cxxopts::value<string>(), "<path>")
         ("exit_after", "Number of seconds to stay alive after last sessions exists", cxxopts::value<int>(), "<duration>")
         ("init_exit_after", "Number of seconds to stay alive at start if no clients connect", cxxopts::value<int>(), "<duration>")
-        ("files", "Files to load", cxxopts::value<vector<string>>(positional_arguments));
+        ("files", "Files to load", cxxopts::value<vector<string>>(positional_arguments))
+        ("idle_session_timeout", "Number of seconds to stay alive after receiving the last message from the frontend", cxxopts::value<int>(), "<duration>");
```

Comment:
The description and argument name should be edited for compatibility with Angus' option PR, but I guess this depends on which PR is going to be merged first.